DependencyInjection/Configuration.php

```diff
@@ -132,7 +132,11 @@ public function getConfigTreeBuilder()
         $rootNode = $builder->root('hwi_oauth');
         $rootNode
             ->children()
-                ->scalarNode('firewall_name')->isRequired()->cannotBeEmpty()->end()
```

Comment:
TBH. I'm not sure we should do this at this point, IMO correct would be both versions, and just `@todo` or something for future release.